### PR TITLE
documentation: using cypress

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Creates new Cypress documentation.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

### CODE REVIEW

File: base/cypress.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Cypress
+
+Cypress is a next generation front end testing tool built for the modern web. Complete documentation can be found [here](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell).
+
+This tool is integrated to RERO-ILS and the following documentation explains how to use it.
+
+## Prerequisites
+
+`poetry run bootstrap` and `poetry run setup`
+Local server (localhost:5000) must be running and the database must contain data, as Cypress will use them to run the tests.
```
Comment (blankoworld) [inline base/cypress.md]: Can you use `*` to create a list?

File: base/cypress.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### Command line
+
+`poetry run cypress`
+A video of the tests is created in folder tests/e2e/cypress/cypress/videos.
+
+### With the test launcher
+
+* `cd tests/e2e/cypress`
+* `npm run cypress` to open Cypress
+* Run the test by clicking on corresponding `spec.js` file
```
Comment (blankoworld) [inline base/cypress.md]: What do you think about adding `*spec.js` or `*.spec.js` instead?

File: base/cypress.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Writing the tests
+
+### Important folders
+
+The base url is stored in cypress.json file
+
+Re-usable code is written here: tests/e2e/cypress/cypress/support/commands.js
```
Comment (blankoworld) [inline base/cypress.md]: Is there any Cyrpess documentation link about this kind of file? (to have more info about that)

File: base/cypress.md (hunk 1 above)
Comment (AoNoOokami) [reply base/cypress.md]: Done

File: base/cypress.md (hunk 2 above)
Comment (AoNoOokami) [reply base/cypress.md]: Done

File: base/cypress.md (hunk 3 above)
Comment (AoNoOokami) [reply base/cypress.md]: Done

**Review by blankoworld — APPROVED** [review]

**Review by benerken — APPROVED** [review]

**Review by iGormilhit — COMMENTED** [review]
Thanks for this! Maybe an entry in `README.md#base` could be a good idea.

**Comment by AoNoOokami** [comment]
> Thanks for this! Maybe an entry in `README.md#base` could be a good idea.

@iGormilhit Done

**Pull request merged**